src/main/java/uk/gov/hmcts/juror/api/moj/controller/response/JurorDetailsCommonResponseDto.java

```diff
@@ -137,7 +137,9 @@ public class JurorDetailsCommonResponseDto {
     @Schema(description = "check for juror manually created")
     private boolean manuallyCreated;
 
-    private boolean hasSummonsResponse;
+    @JsonProperty("response_entered")
+    @Schema(name = "Response Entered", description = "Flag to indicate if a response has been entered for the juror")
+    private Boolean responseEntered;
```

Comment:
Can this be lower case boolean (so it defaults to false)

Reply:
updated